schemas/com.snowplowanalytics.snowplow/media_player_ad_break/jsonschema/1-0-0

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity, shared with all media_player_ad events belonging to the ad break.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_ad_break",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "name": {
+      "description": "Ad break name (e.g., pre-roll, mid-roll, and post-roll).",
+      "type": [
+        "null",
+        "string"
+      ],
+      "maxLength": 4096
+    },
+    "breakId": {
+      "type": "string",
+      "format": "uuid",
+      "description": "An identifier for the ad break."
+    },
+    "startTime": {
+      "type": "number",
+      "description": "Playback time at the start of the ad break.",
```

Comment:
Can we specify a unit here? In context it's obviously seconds, but we also have `timeToStart` below that's milliseconds so probably good to be explicit.

Reply:
Good point, added units!